### PR TITLE
Fix vkCmdDraw*AMD text bugs

### LINKED ISSUE #360: vkCmdDrawIndexedIndirectCountAMD indirect version of ...
> `vkCmdDrawIndexedIndirectCountAMD` behaves similar to `vkCmdDrawIndirectCountAMD` except that the draw count is read by the device from a buffer during execution.

That is not the difference between those two commands.

Also "similar to".

Similar error as was for the `vkCmdDrawIndexedIndirect` in #315.

### PR DESCRIPTION
**Pull request by krOoze**
Fixes #360 and #364.

### CODE REVIEW

**Comment by oddhack** [comment]
@drakos-amd please review and sign off, since it's your extension.

File: doc/specs/vulkan/chapters/pipelines.txt [hunk 1]
```diff
@@ -1502,8 +1502,7 @@ bind point.
 The pipeline bound to ename:VK_PIPELINE_BIND_POINT_COMPUTE controls the
 behavior of flink:vkCmdDispatch and flink:vkCmdDispatchIndirect.
 The pipeline bound to ename:VK_PIPELINE_BIND_POINT_GRAPHICS controls the
-behavior of flink:vkCmdDraw, flink:vkCmdDrawIndexed,
-flink:vkCmdDrawIndirect, and flink:vkCmdDrawIndexedIndirect.
+behavior of all ftext:vkCmdDraw* commands.
```
Comment (TobiasHector) [inline doc/specs/vulkan/chapters/pipelines.txt]: Minor: Might be better to do <<draw, draw commands>>? We use that elsewhere.

Comment (drakos-amd) [reply doc/specs/vulkan/chapters/pipelines.txt]: Agree with Tobias' suggestion, otherwise this PR looks good to me.

Comment (krOoze) [reply doc/specs/vulkan/chapters/pipelines.txt]: So be it!

**Comment by drakos-amd** [comment]
This looks good to me.

**Pull request merged**